### PR TITLE
add labels and attachable properties to network

### PR DESCRIPTION
**Pull request by fengxx**
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/844)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/command/CreateNetworkCmd.java [hunk 1]
```diff
@@ -54,6 +60,11 @@
 
     CreateNetworkCmd withEnableIpv6(boolean enableIpv6);
 
+
+    CreateNetworkCmd withAttachable(boolean attachable);
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/CreateNetworkCmd.java]: please use `Boolean`

File: src/main/java/com/github/dockerjava/api/model/Network.java [hunk 2]
```diff
@@ -65,6 +71,14 @@ public Ipam getIpam() {
         return options;
     }
 
+    public boolean isAttachable() {
+        return attachable == null ? false : attachable;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/Network.java]: please return raw value, on end-user code you can wrap it simply with `BooleanUtils`
null in return value means that there is no value. No value != false from req/rsp view point.

**Comment by KostyaSha** [comment]
please ADD since 1.24 (?) annotations for fields

**Comment by fengxx** [comment]
Labels for network was added in 1.24, can be viewed on https://github.com/moby/moby/blob/master/docs/api/v1.24.md.
I am not sure about "Attachable",  checked 1.18 -> 1.24, no mention in the doc, but python-api is using it, and it is only required when create overlay network in swarm cluster.

**Pull request merged**